src/main/java/racing/Operator.java

```diff
@@ -0,0 +1,20 @@
+package racing;
+
+import java.util.Random;
+
+public class Operator {
+
+    private Random random;
```

Comment:
변하지 않을 값이니 `final` 선언을 해주는건 어떨까요?

```suggestion
    private final Random random;
```